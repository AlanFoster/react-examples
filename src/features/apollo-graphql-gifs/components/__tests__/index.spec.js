/* eslint-env browser */
import React from 'react';
import ImageList from '../';
import renderer from 'react-test-renderer';

it('renders the image list', () => {
  const component = renderer.create(<ImageList />);

  expect(component).toMatchSnapshot();
});
