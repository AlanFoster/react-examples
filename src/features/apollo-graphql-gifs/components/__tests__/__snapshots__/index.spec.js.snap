// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the image list 1`] = `
<div>
  hello world!
</div>
`;
